community/primitive-collections/src/main/java/org/neo4j/collection/primitive/PrimitiveIntCollections.java

```diff
@@ -750,4 +754,50 @@ public static PrimitiveIntSet asSet( int[] values )
         }
         return set;
     }
+
+    /**
+     * Adds all the items in {@code iterator} to {@code collection}.
+     * @param <C> the type of {@link Collection} to add to items to.
+     * @param iterator the {@link Iterator} to grab the items from.
+     * @param collection the {@link Collection} to add the items to.
+     * @return the {@code collection} which was passed in, now filled
+     * with the items from {@code iterator}.
+     */
+    public static <C extends Collection<Integer>> C addToCollection( PrimitiveIntIterator iterator, C collection )
+    {
+        while ( iterator.hasNext() )
+        {
+            collection.add( iterator.next() );
+        }
+        return collection;
+    }
+
+    public static List<Integer> asBoxedList( PrimitiveIntIterator iterator )
```

Comment:
It's just that it clashes with IteratorUtil method(s)... but we can qualify those methods instead